packages/account-v2/.eslintrc.js

```diff
@@ -0,0 +1,81 @@
+module.exports = {
+    env: { browser: true, es6: true, jest: true },
+    extends: [
```

Comment:
I think we should just extend `@deriv/eslint-config-deriv`, it has all the below configs - https://github.com/binary-com/eslint-config-deriv/blob/main/index.js

Reply:
cc: @ali-hosseini-deriv

Reply:
The lib `@deriv/eslint-config-deriv` is extended. Some of the rules used are not present in @deriv/eslint-config-deriv, hence they were listed. camelCase is a duplication. That can be removed